autogpt_platform/frontend/public/mockServiceWorker.js

```diff
@@ -8,7 +8,7 @@
  * - Please do NOT serve this file on production.
```

Comment:
not part of your PR but do you know how we're handling this?

Reply:
I don't know and I raised that issue on Discord